src/hooks/useSymbolList.ts

```diff
@@ -0,0 +1,66 @@
+import { getDataFromDB } from '@/lib/indexDB';
+import { useQuery } from '@tanstack/react-query';
+import { useEffect, useState } from 'react';
+
+export type SymbolList = {
+  symbol: string;
+  exchange: string;
+  exchangeShortName: string;
+  price: string;
+  name: string;
+};
+
+const api_FMP_key = import.meta.env.VITE_FMP_API_KEY;
+
+export const fetchSymbolList = async (): Promise<SymbolList[]> => {
+  const response = await fetch(
```

Comment:
Instead of `fetch` we should use [ky](https://github.com/sindresorhus/ky), create and configure an api client and use that everywhere in our application.  Here is an example how it can be setup. It should live somewhere in global folders, then imported into features.

```ts
import ky from "ky"

import { env } from "@/config/env"

export const api = ky.create({
  prefixUrl: env.VITE_BE_API_URL,
  headers: {
    Accept: "application/json",
    "Content-Type": "application/json",
  }
})

```